### DISCUSSION TITLE
截天帝论坛 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#206] [模板讨论]
截天帝论坛 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by 2083779614** [comment]
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://jietiandi.net/plugin.php?id=zqlj_sign报错无法正常签到

**Reply by lpx2223** [reply]
之前一直可以，但是删除重新新建一个签到后，就不行了，出现
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://jietiandi.net/plugin.php?id=zqlj_sign
的错误，求救

**Comment by miaoxiaozhao** [comment]
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://jietiandi.net/plugin.php?id=zqlj_sign
